src/LabeledTextItem/LabeledTextItem.less

```diff
@@ -8,7 +8,7 @@
 
 	// Header
 	.moon-labeledtextitem-header {
-		display: inline-block;
+		display: block;
```

Comment:
Changing this pushes the arrow for the expandable pickers all the way to the right. Was that your expectation? If so, we should probably document that as such in the ticket so we acknowledge it was intentional.